app/javascript/packages/document-capture/components/warning.tsx

```diff
@@ -45,6 +60,9 @@ function Warning({
   heading,
   actionText,
   actionOnClick,
+  altActionText,
```

Comment:
just want to make sure i understand - altAction is variant-b's cta?

Reply:
Speaking first generally, we augmented this component to allow for two actions in the same paradigm as you might see an "OK" and a "Cancel". The primary user action is as before: Blue button, white text. We've added an alternative user action option that is presented as a white button with blue outline.

You are correct about the way this is used in the variant B Warning component.

Reply:
ahh ok so then i should think of this as a [stylistic] secondary version of the button.

Reply:
Hm I might be misunderstanding you here - "a [stylistic] secondary version of the button" makes it sound like a different (stylistic) way to present a sole intended user option, like you would be using the white-with-blue-outline style instead of the blue. While the implementation would technically allow for that (I think...), that isn't the intended use. Rather than a different way to display _the same_ button, we wanted the Warning to display two _different_ buttons.